src/Kunstmaan/SeoBundle/Resources/views/SeoTwigExtension/metadata.html.twig

```diff
@@ -2,38 +2,96 @@
 {% if seo.getMetaDescription() %}
     <meta name="description" content="{{ seo.getMetaDescription() }}">
 {% endif %}
-{% if seo.getMetaKeywords() %}
-    <meta name="keywords" content="{{ seo.getMetaKeywords() }}">
-{% endif %}
 {% if seo.getMetaAuthor() %}
     <meta name="author" content="{{ seo.getMetaAuthor() }}">
 {% endif %}
 {% if seo.getMetaRobots() %}
     <meta name="robots" content="{{ seo.getMetaRobots() }}">
+{% else %}
+    <meta name="robots" content="all">
 {% endif %}
-{% if seo.getMetaRevised() %}
-    <meta name="revised" content="{{ seo.getMetaRevised() }}">
-{% endif %}
+
+<!-- Open Graph -->
 {% if seo.getOgType() %}
     <meta property="og:type" content="{{ seo.getOgType() }}">
 {% endif %}
 {% if seo.getOgTitle() %}
     <meta property="og:title" content="{{ seo.getOgTitle() }}">
+{% else %}
+    <meta property="og:title" content="{{ seo.metaTitle }}">
 {% endif %}
 {% if seo.getOgDescription() %}
     <meta property="og:description" content="{{ seo.getOgDescription() }}">
 {% endif %}
 {% if seo.getOgUrl() %}
-   <meta property="og:url" content="{{ seo.getOgUrl() }}">
+    <meta property="og:url" content="{{ seo.getOgUrl() }}">
 {% else %}
-    {% if currentNode %}
-        <meta property="og:url" content="{{ url('_slug', { 'url': currentNode.slug, '_locale': app.request.locale }) }}"/>
-    {% endif %}
+{% if currentNode %}
+    <meta property="og:url" content="{{ url('_slug', { 'url': currentNode.slug, '_locale': app.request.locale }) }}"/>
+{% endif %}
 {% endif %}
 {% if seo.getOgImage() is defined and seo.getOgImage() is not null %}
+    {% if seo.ogImage.contentType == "remote/video" %}
+        {% if seo.ogImage.metaData.type == "youtube" %}
+            {% set videoUrl = "youtube.com/embed/"~seo.ogImage.metaData.code %}
+        {% elseif seo.ogImage.metaData.type == "dailymotion" %}
+            {% set videoUrl = "www.dailymotion.com/embed/video/"~seo.ogImage.metaData.code %}
+        {% else %}
+            {% set videoUrl = "player.vimeo.com/video/"~seo.ogImage.metaData.code %}
+        {% endif %}
+    <meta property="og:video" content="http://{{ videoUrl }}">
```

Comment:
Don't use hardcoded http references ... Either keep the protocol of the current request or omit the protocol